src/pages/aurora-site/aurora-chat/index.jsx

```
@@ -154,121 +159,99 @@ const AuroraChat = () => {
-            <h3 className="text-sm font-medium text-green-800">Use Context</h3>
-            <p className="text-xs text-green-600/80">Pay attention to how words are used in different situations.</p>
+          <div className="p-3 bg-dark-blue-1 rounded-xl border border-dark-blue-4 hover:bg-dark-blue-4 transition-colors">
+            <h3 className="text-sm font-medium text-green">Use Context</h3>
+            <p className="mt-1 text-xs text-neutral-2">Pay attention to how words are used in different situations.</p>
           </div>
-          <div className="p-2.5 bg-purple-50/80 rounded-xl border border-purple-100 hover:bg-purple-50 transition-colors">
-            <h3 className="text-sm font-medium text-purple-800">Review Regularly</h3>
-            <p className="text-xs text-purple-600/80">Go back to previous conversations to reinforce what you've learned.</p>
+          <div className="p-3 bg-dark-blue-1 rounded-xl border border-dark-blue-4 hover:bg-dark-blue-4 transition-colors">
+            <h3 className="text-sm font-medium text-purple-400">Review Regularly</h3>
+            <p className="mt-1 text-xs text-neutral-2">Go back to previous conversations to reinforce what you've learned.</p>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)